indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -474,6 +476,19 @@
   ;; reindex variables associated with the collection
   (reindex-associated-variables context concept-id revision-id))
 
+(defn get-concept-delete-log-string
+  "Get the log string for concept-delete. Appends granules deleted if concept-type is collection"
+  [concept-type context concept-id revision-id all-revisions-index?]
+  (let [log-string (format "Deleting concept %s, revision-id %s, all-revisions-index? %s"
+                           concept-id 
+                           revision-id 
+                           all-revisions-index?)]
+    (if (= concept-type :collection)
+      (format "%s. Removing %d granules."
+              log-string
+              (search/find-granule-hits context {:collection-concept-id concept-id}))
+      log-string)))
+
```

Comment:
Just sth minor - if this function is only used in this file, then make it private - (defn- .